packages/shared/src/utils/validation/declarative-validation-rules.ts

```diff
@@ -112,6 +113,10 @@ const initPreBuildDVRs = () => ({
         func: validLetterSymbol,
         message: form_error_messages.letter_symbol,
     },
```

Comment:
Can you check on the `letter_symbol` object type, if there's no other place using it, you may remove it from the code base.